pkg/webhook/webhook.go

```diff
@@ -832,3 +894,13 @@ func SetInjectHugepageDownApi(hugepageFlag bool) {
 func SetHonorExistingResources(resourcesHonorFlag bool) {
 	honorExistingResources = resourcesHonorFlag
 }
+
+// SetCustomizedInjections sets additional injections to be applied in Pod spec
+func SetCustomizedInjections(injections *corev1.ConfigMap) {
```

Comment:
shouldn't we also remove stale entries from customizedInjections Map (to handle delete cases in config map) ?

Reply:
Yes, we should.
Done